app/pages/preview/template.test.js

```diff
@@ -7,125 +7,32 @@ const setup = {
-          },
-        },
-      };
-
-      harness.request(context, ($) => {
-        expect($('[data-test-id="view-integrations"]').length).toEqual(1);
-        expect($('[data-test-id="view-question-data-text-link-supplier-integrations"]').length).toEqual(1);
-      });
-    }));
-
-    it('should render the integrations section document-link is provided', createTestHarness(setup, (harness) => {
-      const context = {
-        sections: {
-          integrations: {
-            answers: {
-              'document-link': 'document.pdf',
-            },
-          },
-        },
-      };
-
-      harness.request(context, ($) => {
-        expect($('[data-test-id="view-integrations"]').length).toEqual(1);
-        expect($('[data-test-id="view-question-data-text-link-authority-integrations"]').length).toEqual(1);
-      });
-    }));
-  });
-
-  it('should render the client application types section when provided', createTestHarness(setup, (harness) => {
-    const context = {
-      sections: {
-        'client-application-types': {},
-      },
-    };
-
-    harness.request(context, ($) => {
-      expect($('[data-test-id="view-client-application-types"]').length).toEqual(1);
-    });
-  }));
-
-  it('should render the hosting types section when one of the sections is provided', createTestHarness(setup, (harness) => {
-    const context = {
-      sections: {
-        'hosting-type-private-cloud': {},
-      },
-    };
-
-    harness.request(context, ($) => {
-      expect($('[data-test-id="view-hosting-types"]').length).toEqual(1);
+      const backLink = $('[data-test-id="view-solution-page-back-link"] a');
+      expect(backLink.length).toEqual(1);
+      expect(backLink.text().trim()).toEqual('Go back to previous page');
     });
   }));
 
-  it('should render the capabilities section when provided', createTestHarness(setup, (harness) => {
+  it('should render the viewSolution component', createTestHarness(setup, (harness) => {
     const context = {
-      sections: {
-        capabilities: {
-          answers: {
-            'capabilities-met': [{
-              name: 'Communicate With Practice - Citizen',
-              version: '1.0.1',
-              description: 'Supports secure and trusted electronic communications between Citizens and the Practice. Integrates with Patient Information Maintenance.',
-              link: 'https://gpitbjss.atlassian.net/wiki/spaces/GPITF/pages/1391134188/Communicate+With+Practice+-+Citizen',
-            }],
-          },
-        },
+      solutionHeader: {
+        id: '100000-001',
+        name: 'Write on Time',
+        supplierName: 'Really Kool Corporation',
+        isFoundation: true,
+        lastUpdated: '1996-03-15T10:00:00',
       },
+      returnToDashboardUrl: '/supplier/solution/100000-001',
     };
 
     harness.request(context, ($) => {
-      expect($('[data-test-id="view-capabilities"]').length).toEqual(1);
+      expect($('h1').text().trim()).toEqual('Write on Time');
```

Comment:
Still not sure about this. Can we just check the view-solution component is rendered. Rather than it's innards

Reply:
Ok as we check the 'innards' in the integration tests :)